ruamel_yaml/meta.yaml

```diff
@@ -2,17 +2,19 @@ package:
   name: ruamel_yaml
   version: 0.11.14
 
+source:
+  fn: ruamel_yaml-0.11.14.tar.gz
+  url: https://bitbucket.org/ruamel/yaml/get/0.11.14.tar.gz
+  sha256: b78d394e192b6d0911bd59f794ceea88143fac4e373b76ed7e48143a2a29b33c
+
 build:
   number: 0
   no_link: .*\.(pyd|dll)                       [win]
 
-source:
-  hg_url: https://bitbucket.org/ruamel/yaml
-  hg_tag: 0.11.14
-
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
Should note that as the `setup.py` file we replace the existing one with does not require `setuptools`, we do not need to add `--single-version-externally-managed --record=record.txt`.